src/Classes/ServiceAPI/MyRadio_Swagger2.php

```diff
@@ -348,6 +370,44 @@ private static function getTags($apis)
         return $tags;
     }
 
+    private static function getMethodOpType($method)
+    {
+        $name = $method->getName();
+
+        //Note the ordering is important - create is static!
+        if (
+            CoreUtils::startsWith($name, 'create') ||
+            CoreUtils::startsWith($name, 'add')
+        ) {
```

Comment:
True, but I think it looks better if ifs are just split over 2 lines (with an extra indent for the additional line), rather than spread over several.

I'll concede the function call though